src/ROUTES.ts

```diff
@@ -585,6 +585,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/tags/edit',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/tags/edit` as const,
     },
+    WORKSPACE_TAG_EDIT: {
+        route: 'workspace/:policyID/tags/:tagName/edit',
+        getRoute: (policyID: string, tagName: string) => `workspace/${policyID}/tags/${encodeURI(tagName)}/edit` as const,
```

Comment:
`encodeURI` encodes all values except for `/` which will be a problem in the URL. Given that tags can have `/` in the name, we should use `encodeURIComponent`
```suggestion
        getRoute: (policyID: string, tagName: string) => `workspace/${policyID}/tags/${encodeURIComponent(tagName)}/edit` as const,
```